src/components/media-player/media-player.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MediaPlayerModule } from './media-player.module';
+
+@Component({
+    selector: 'ux-media-player-audio',
+    template: `<ux-media-player *ngIf="type === 'audio'" class="media-player" type="audio" [displayName]="displayName" [source]="audioSource"></ux-media-player>
```

Comment:
I don't think we need the `ngIf` here? also we have a `type` and `mode` variable in the component class which is never used.